src/main.c

```diff
@@ -43,7 +43,7 @@
 #include "privs.h"
 
 #ifdef PR_USE_OPENSSL
-# include <openssl/opensslv.h>
+# include <openssl/crypto.h>
```

Comment:
The _ order_ in which OpenSSL pulls in its header files is important; getting it wrong (as we were) would not properly set the build-time defines (such as `#define OPENSSL_FIPS`), and thus we might not properly detect FIPS support.